README.md

```diff
@@ -185,7 +184,7 @@ On all distributions please use -DUSE_SHARED_LIBS=OFF to statically link zlib, l
 #### Libraries
 * libuuid
 * librocksdb (built and statically linked)
-* libcurl-openssl (If not available or desired, NSS will be used)
+* libcurl-openssl
```

Comment:
```suggestion
* libcurl-openssl (built and statically linked)
```

Reply:
Updated in 6c350a979f1fbdd0e2e020feeb0ad7087b64771c

Reply:
did @szaszm mean line 186 (as well)?

Reply:
Good point, actually it seems that a lot of build and runtime dependencies are not relevant and should be updated at a couple of places. I created a separate jira ticket for this: https://issues.apache.org/jira/browse/MINIFICPP-2331 and additionally removed the unneeded build and runtime system dependencies here in 96d3b6a6896c37bb5b493a035a6f258e1bc97ada

Reply:
I didn't notice line 186, but the same applies there.
We shouldn't remove the listed dependencies, even if a user doesn't need to install them. Those that don't need to be installed used to be marked with `(built and statically linked)`, meaning no system version is necessary, but users know that code from those libraries is used. If you want to remove them from this list, then we need to add them somewhere else.
If you want to do the addition of a second list in MINIFICPP-2331, then you should also do the removal from the existing list there. I understand that the list is not perfect at the moment, but I believe this half-change in 52c35a6334740c874f3515e3e95448eaeca8f143 makes it worse by removing even more information.

I see a few valid options for us in the context of this PR:
1. Mark openssl and curl as `(built and statically linked)`, and not touch the rest of the entries
2. Same as 1, plus do the same for other entries that have been recently changed. If something is now built and statically linked by our build system, mark it as such. If something is no longer used, remove it.
3. Do all changes described in [MINIFICPP-2331](https://issues.apache.org/jira/browse/MINIFICPP-2331) that are relevant for these two lists.

Reply:
After a discussion, we agreed to keep all "built and statically linked" entries removed, but add a note that these are only the libraries used from the system, and to add a reference to the NOTICE file for the full list of dependencies for any interested users.

Reply:
Added a separate section with link to the LICENSE and NOTICE files to find information on the bundled thirdparty components and clarified the list of required system packages in b1d87f4a5ce46fb23bd23ded6775b62279314d72